### PR TITLE
Scaffold a guide for GraphQL

### PR DESCRIPTION
**Pull request by lancejjohnson**
thoughtbot seems to be doing more work with GraphQL for client projects.
We would like to be to discuss and document what we discover as best
practices, recommended solutions, and issues to avoid for the benefit of
thoughtbot as we go. This commit provides an initial scaffolding for a
guide to GraphQL. Future commits can begin to fill in details.

### CODE REVIEW

File: graphql/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# GraphQL at thoughtbot
```
Comment (jferris) [inline graphql/README.md]: I would remove the "at thoughtbot" instances. The guide is under the thoughtbot org, and I don't believe these approaches are specific to thoughtbot's needs.

Comment (lancejjohnson) [reply graphql/README.md]: 👍

**Comment by mike-burns** [comment]
I enjoyed:

https://github.com/Shopify/graphql-design-tutorial/blob/master/TUTORIAL.md
https://graphql-ruby.org/mutations/mutation_errors#errors-as-data

Also it took Wenger to point this out to me: GraphQL Relay is not the relay library but instead a useful thing to know about GraphQL.

**Comment by mike-burns** [comment]
But also, this is great, thank you.

**Pull request merged**